src/utils/format/index.ts

```diff
@@ -16,3 +16,10 @@ export const formatDate = (
 export const toLocaleCurrency = (value: number): string => {
   return value.toLocaleString('kr')
 }
+
+export const toQueryString = (value: {
+  [key: string]: string | number
+}): string =>
```

Comment:
```suggestion
export const toQueryString = (value: Record<string, string | number>): string =>
```
유틸리티 타입써서 표현해도 될 것 같습니다!

Reply:
요거 린트에서 위 상태로 fix해주네용 ! 룰 변경을 따로 해야할 것 같아용

Reply:
앗 그러면 지금 그냥 그대루 가시쥬!!